runbook/mapping/es.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2016: Mirantis Inc.
```

Comment:
Let's move this module to: "runbook/storage.py" I like how it's done in Availability service (and actually we should do the same in health service)